x-pack/plugins/apm/public/components/shared/time_comparison/index.tsx

```diff
@@ -59,13 +60,15 @@ function getSelectOptions({ start, end }: { start?: string; end?: string }) {
     text: formatPreviousPeriodDates({ momentStart, momentEnd }),
```

Comment:
Move `prevPeriodOption` down to the bottom. No need to calculate this if it's not necessary.